io.github.electron.geforcenow.yaml

```diff
@@ -0,0 +1,53 @@
+app-id: io.github.electron.geforcenow
+branch: stable
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '20.08'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node14
+separate-locales: false
+command: start-geforcenow-electron
+
+finish-args:
+  - '--share=ipc'
+  - '--socket=x11'
+  - '--socket=wayland'
```

Comment:
Does it actually work under Wayland or fallbacks to xwayland?

Reply:
Yes, the application works under wayland.